cc/main.cc

```diff
@@ -65,6 +66,8 @@ DEFINE_uint64(seed, 0,
               "Random seed. Use default value of 0 to use a time-based seed. "
               "This seed is used to control the moves played, not whether a "
               "game has resignation disabled or is a holdout.");
+DEFINE_bool(even, true,
```

Comment:
I don't really know what `even` means in this context. Can you choose a more descriptive name or alternatively just remove the flag altogether; I'm not sure there's ever a realistic reason to set it to false is there?

Reply:
Agreed, removed.